workflow-Estimate/workflow/config/workflow.ini

```diff
@@ -0,0 +1,35 @@
+#Output file configuration
+library=library
+run=run
+
+output_dir=seqware-results
+output_prefix=./
+manual_output=false
+tmp_dir=tmp
+
+
+#Programs
+rpath=/.mounts/labs/PDE/Modules/sw/R/R-3.3.0
+rLib=/.mounts/labs/gsiprojects/gsi/tools/R_libs
+
+#memory
+estimate_mem=20
```

Comment:
I suggest naming this with "mem_mb" or "mem_gb" to make it clear the what units the memory is in.